SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestHandoverReplication/SpatialTestHandoverActorComponentReplication.h

```diff
@@ -20,6 +20,37 @@ enum class EHandoverReplicationTestStage
 	Final,
 };
 
+USTRUCT()
+struct FHandoverReplicationTestStructInner
+{
+	GENERATED_BODY()
+
+	UPROPERTY(Handover)
```

Comment:
I hate handover so much - we should have just used `Replicated` with a special condition.